packages/python/javdb_spider/fetch/login_coordinator.py

```
@@ -130,12 +167,384 @@ def __init__(
+                    self._cooldown_until_ms > 0
+                    and now_ms >= self._cooldown_until_ms
+                ):
+                    drained = list(self._pending_login_tasks)
+                    self._pending_login_tasks.clear()
+                    self._cooldown_until_ms = 0
+                    logger.info(
+                        "P2-C login cooldown lifted — re-dispatching %d "
+                        "parked task(s) to retry login",
+                        len(drained),
+                    )
+                    for proxy_name, task, login_queue in drained:
```

Ruff (0.15.12)

[warning] Loop control variable `proxy_name` not used within loop body

(B007)